src/gaxios.ts

```diff
@@ -55,7 +74,6 @@ function loadProxy() {
   }
   return proxy;
 }
```

Comment:
@bcoe, I want to ask about the default behavior. If a URL comes in that is not listed under no_proxy or proxy, should it be proxied? Currently that's the way it is with the code as it stands now, but I feel like the default should be the other way around. In other words, we should maybe do some kind of checking to see if the URL equals any of the HTTPS_PROXY env variables.